### PR TITLE
FWI-3290 - [CI Plugin] Ci plugin must use Insights Repository Report Config

### PR DESCRIPTION
**Pull request by vitorvezani**
This PR fixes #

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### What's the goal of this PR?
Support reports config (`enabled`/`disabled`) when using auto-discovery mode

### What alternative solution should we consider, if any?

[Internal Ticket FWI-3290](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-3290)

### CODE REVIEW

File: plugins/ci/pkg/util/util.go [hunk 1]
```diff
@@ -42,7 +45,7 @@ func RemoveTokensAndPassword(s string) string {
 	// based on x-access-token
 	index := strings.Index(s, "x-access-token")
 	index2 := strings.Index(s, "@github.com")
-	if index > 0 || index2 > 0 {
+	if index > 0 && index2 > 0 {
```
Comment (vitorvezani) [inline plugins/ci/pkg/util/util.go:48]: unrelated fix

File: plugins/ci/pkg/ci/ci.go [hunk 2]
```diff
@@ -634,6 +670,14 @@ func (ci *CIScan) printScannedFilesInfo() {
 			fmt.Printf("\t[%d/%d] - %s/%s\n", i+1, s, h.Path, h.Name)
 		}
 	}
+
+	s = len(ci.config.Terraform.Paths)
```
Comment (vitorvezani) [inline plugins/ci/pkg/ci/ci.go:674]: unrelated fix

File: plugins/ci/pkg/ci/auto_detection.go [hunk 3]
```diff
@@ -106,11 +106,6 @@ func ConfigFileAutoDetection(basePath string) (*models.Configuration, error) {
 			Paths: terraformPaths,
 		},
 	}
-	if len(terraformPaths) > 0 {
```
Comment (vitorvezani) [inline plugins/ci/pkg/ci/auto_detection.go:109]: @ivanfetch I might have missed the point of having it being enabled here... please, let me know the use-case

**Review by jdesouza — APPROVED** [review]
🚀
